---
layout: post
title: DNA Isolation from frozen Oly larvae
---

It is happening. My trial last week worked sufficiently well in the samples whic started with ~30ug tissue, so I'm going to move forward with the actual extraction.  As per Sam's suggestion, I read the MethylMiner kit instructions to see what finished product I'll need for the DNA methylation enrichment step.  Here's what I learned: 

  * The kit provides materials for 25 affinity-based separations when starting with 5 ng–1 μg of fragmented genomic input DNA, and is scalable up to a single separatation using 25 μg of input DNA. The methlyated DNA may be eluted into as many as 8 fractions per separation.  
  * For downstream analyses like PCR and qPCR, as little as 5 ng of input DNA can be used. **For applications that require larger amounts of methylated DNA, such as library construction for high-throughput sequencing or amplification and labeling for microarray analysis, starting amounts of 10–25 μg of fragmented input DNA are most appropriate**, though in some cases as little as 1 μg can be used. Typical total yields of mammalian CpG-methylated DNA are 3–20% of the input mass of DNA, or 0.3–5.0 μg when starting with 10–25 μg. 
  * DNA may be fragmented using your method of choice. DNA must be fragmented to an average size of less than 1,000 bp and should be in DNasefree water, TE buffer, or another low ionic-strength, neutral pH buffer. The fragment size should be appropriate for your downstream analysis. For example, DNA fragmented to an average length of ~250 bp is suitable for assay by real-time quantitative PCR (qPCR). Similarly, DNA fragmented to an average length of ~100–200 bp is suitable for fragment library construction for short-read high-throughput sequencing.  
  * For input amounts <1ug, add 5 ng–1 μg, added volume should be ≤ 80 μl 
  * For input amounts 1 μg-10μg, the fragmented DNA should be at a concentration of 25 ng/μl or higher; added volume will be 40–400 μl.   
  * **For input amounts 10-25ug,  the fragmented DNA should be at a concentration of 25 ng/μl; added volume will be 400–1000 μl.**   

Since I need to shoot for at least 10ug DNA per sample (which is a lot), I will do the following: 
  * Increase my initial tissue mass to 50mg, which is the max I can do for each column.  
  * Extract 3 replicates from each tissue sample; I should have plenty of larvae in each sample to do so.  
  * I should get ~3ug DNA per sample, given the yield from test run. 
  * For final elution volume, I should not exceed 300uL.  
  
